Bitbucket-Pipelines/two-stage-pipeline-template/{{cookiecutter.outputDir}}/bitbucket-pipelines.yml

```diff
@@ -4,11 +4,16 @@ pipelines:
   branches:
     feature:
       - step:
+          oidc: true
           name: Build and Package
           script:
             - export SAM_TEMPLATE="{{ cookiecutter.sam_template }}"
-            - export PIPELINE_USER_ACCESS_KEY_ID=${{ "{" }}{{cookiecutter.pipeline_user_aws_access_key_id_variable_name}}{{ "}" }}
-            - export PIPELINE_USER_SECRET_ACCESS_KEY=${{ "{" }}{{cookiecutter.pipeline_user_aws_secret_access_key_variable_name}}{{ "}" }}
+            - export PERMISSIONS_PROVIDER="{{ cookiecutter.permissions_provider }}"
+            - |
+              if [ "$PERMISSIONS_PROVIDER" = IAM ]; then
```

Comment:
Can this be done through Jinja instead? A little ambivalent, but the branches add a lot of cognitive complexity, and as a customer I'd much rather get a nicely tidied pipeline definition that matches what I requested, rather than an amalgamation of different combinations.

Reply:
Thanks for pointing this out, I hadn't realized I could use Jinja to accomplish the same thing